2021/27xxx/CVE-2021-27853.json

```diff
@@ -4,15 +4,121 @@
     "data_version": "4.0",
```

Comment:
Invalid source.

Reply:
I believe this is `references/reference_data/*/refsource`  issue? Should be fixed if that is the case.